cmd/node/main.go

```diff
@@ -963,8 +964,23 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		return err
 	}
 
+	epochStartMetaBlockInterceptor, err := interceptors.NewEpochStartMetaBlockInterceptor(
+		interceptors.ArgsEpochStartMetaBlockInterceptor{
+			Marshalizer:               coreComponents.InternalMarshalizer,
+			Hasher:                    coreComponents.Hasher,
+			NumConnectedPeersProvider: networkComponents.NetMessenger,
+			ConsensusPercentage:       50,
```

Comment:
You can define in core/constants.go this constant and use it here

Reply:
done